server/autolinkplugin/command.go

```diff
@@ -12,7 +12,8 @@ import (
 	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
 )
 
-const template, pattern = "Template", "Pattern"
+const optName, optTemplate, optPattern, optScope, optDisabled, optDisableNonWordPrefix, optDisableNonWordSuffix, optWordMatch = "Name", "Template", "Pattern", "Scope", "Disabled", "DisableNonWordPrefix", "DisableNonWordSuffix", "WordMatch"
+const command = "/autolink"
```

Comment:
0/5 make it a bit more specific, intention-revealing; maybe `autolinkCommand` or `cmdAutolink`?